ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -791,6 +800,96 @@ private void SensorMonitor()
                 azTempSafe = checkTemp(azTemp, azTempSafe);
                 elTempSafe = checkTemp(elTemp, elTempSafe);
 
+                // Sensor status routine, checks for each sensor to update the status in the DB
+                // Check Gate
+                sensors.gate = (SByte)SensorStatusEnum.NORMAL;
+                // if gate open
+                // sensors.gate = (SByte)SensorStatusEnum.ALARM;
+
+                // Check azimuth temp 1
+                sensors.az_motor_temp_1 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.AzimuthTemperature1Status;
+
+                // Check azimuth temp 2
+                sensors.az_motor_temp_2 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.AzimuthTemperature2Status;
+
+                // Check elevation temp 1
+                sensors.el_motor_temp_1 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.ElevationTemperature1Status;
+
+                // Check elevation temp 2
+                sensors.el_motor_temp_2 = (SByte)RadioTelescope.SensorNetworkServer.SensorStatuses.ElevationTemperature2Status;
```

Comment:
The sensor statuses for the sensor network sensors are indexed differently, so with this code temperature status could be reading `OK` but would be assigned `Warning` because `OK` is 1 in the sensor network enum.

- [ ] Fixed sensor enum conversion